di-ipv-core-stub/README.MD

```diff
@@ -17,29 +17,30 @@ This stub allows us to:
 ## Config
 
 Core Stub config from the `di-ipv-config` repository directory `/stubs/di-ipv-core-stub` will be available at:
-* Docker image: `/app/config/`
+* Docker image: `/app/`
```

Comment:
Rather than create an empty config directory, it seemed easier to mount the required files into the existing app directory.